modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -0,0 +1,18 @@
+[[delete.clients]]
+= Delete Clients
+
+If you need to remove a client from your {productanme} Server, you can use the {webui} to delete it.
+This procedure works for both traditional and Salt clients, however Salt clients require extra steps to clean up configuration files after the client is deleted.
```

Comment:
Later on, you say that the server tries to do this, and that sounds as if the server is sometimes able to do this without further user action.

Reply:
Heh, this is the problem with passive voice :wink:

Reply:
```suggestion
This procedure works for both traditional and Salt clients.
```